src/Lecture1.hs

```diff
@@ -54,7 +55,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Num a => a -> a -> a
+sumOfSquares x y = x*x+y*y
```

Comment:
Usually, you're encouraged to put spaces between operators to make different parts of the code more visible 👾 

```suggestion
sumOfSquares x y = x * x + y * y
```